scripts/plotParticles.py

```diff
@@ -18,10 +18,10 @@ def particleplotting(filename,tracerfile, mode):
     fig, ax = plt.subplots()
     if tracerfile != 'none':
       tfile = Dataset(tracerfile,'r')
-      X = tfile.variables['x']
-      Y = tfile.variables['y']
-      P = tfile.variables['P']
-      plt.contourf(np.squeeze(X),np.squeeze(Y),np.squeeze(P))
+      X = tfile.variables['xu_ocean']
+      Y = tfile.variables['yu_ocean']
```

Comment:
This will most likely break plotting for other grid files that follow a different convention. We need to make this more generic and configurable, possibly even using our own `Grid` or `Field` classes.